src/utilities/printf.c

```diff
@@ -129,12 +129,24 @@ free_format( char *newformat )
 HYPRE_Int
 hypre_ndigits( HYPRE_BigInt number )
```

Comment:
Why not just have this function return a call to `hypre_ndigits_ull` with an argument cast?